dts/bindings/mmc/mmc-spi-slot.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2018 Google LLC.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: MMC/SD/SDIO slot connected via SPI
+id: zephyr,mmc-spi-slot
+version: 0.1
+
+description: MMC/SD/SDIO slot connected via SPI
+
+inherits:
+    !include spi-device.yaml
+
+properties:
+    compatible:
+      constraint: "zephyr,mmc-spi-slot"
+
+    gpios:
```

Comment:
I propose to follow the same approach as linux/Documentation/devicetree/bindings/spi/spi-bus.txt. We should add a property cs-gpios in spi.yaml and modify extract_dts_includes.py to support it.

If, for example, the SPI has 2 CS lines, and the cs-gpios property looks like this:

`cs-gpios = <&gpioA 12 0> <&gpioC 11 0>;`

we have the following mapping:
```
cs0 : &gpioA 12 0
cs1 : &gpioC 11 0
```
The 0 in spi-device@0 and property reg = <0> point to cs0.

Reply:
@ydamigos that sounds fine.  Who should I ask to do the dts generator changes?